src/index.ts

```
@@ -247,9 +249,9 @@ module.exports = {
   },
 
   rules: {
+    'no-excessive-parent-chain': noExcessiveParentChain,
     'prefer-document-flattening': preferDocumentFlattening,
-    'prefer-block-comments-for-declarations':
-      preferBlockCommentsForDeclarations,
+    'prefer-block-comments-for-declarations': preferBlockCommentsForDeclarations,
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)